packages/cfd/src/Containers/mt5-migration-modal/mt5-migration-account-icons.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Jurisdiction, JURISDICTION_MARKET_TYPES } from '@deriv/shared';
+
+type TMT5MigrationAccountIconsProps = {
+    to: string;
+    type: string;
+};
+
+const MT5MigrationAccountIcons = observer(({ to, type }: TMT5MigrationAccountIconsProps) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    const getFromAccountIcon = () => {
+        switch (type) {
+            case JURISDICTION_MARKET_TYPES.DERIVED:
+                return 'IcMt5SvgDerived';
+            case JURISDICTION_MARKET_TYPES.FINANCIAL:
+                return 'IcMt5SvgFinancial';
+            default:
+                return '';
+        }
+    };
+
+    const getToAccountIcon = (to_type: string) => {
```

Comment:
can we cook `to_type` here inside the function, instead of buying it as an argument?
`const to_type = `${to}_${type}``